addons/web_tour/static/src/js/tip.js

```diff
@@ -155,25 +250,49 @@ var Tip = Widget.extend({
         if (rtlMap[appendAt] && _t.database.parameters.direction === 'rtl') {
             appendAt = rtlMap[appendAt];
         }
-        this.$el.position({
-            my: this._get_spaced_inverted_position(appendAt),
-            at: appendAt,
-            of: this.$anchor,
-            collision: "none",
-        });
+
+        // Get the correct tip's position depending of the tip's state
+        let $parent = this.$ideal_location;
+        if ($parent.is('html,body') && this.viewPortState !== "in") {
+            this.$el.css({position: 'fixed'});
+        }
```

Comment:
Why do we still need a position fixed tip ?

Reply:
I don't think it was the only bug I had when I tried to avoid that, but I remember this one :
When the tip is linked to the body, because the body doesn't move bottom with the scroll, the tip wasn't able to come down when I scrolled down.

And @JKE-be advise me to keep that fixed position too.

Reply:
I suppose `position: fixed` can solve the problem you mention since it is the window which scrolls and not the body but it will break in all other cases: the scrollable element might be in something that scrolls so `position: fixed` is not a solution. I don't really mind it but I think you will avoid to change the line below (offset = ...) if you don't use `position: fixed` ?

Reply:
Indeed, if there is a scroll in another one, it will break if the use scroll the parent one. Depends if we have any case like that or not.